code/datums/jobs.dm

```diff
@@ -138,6 +138,51 @@
 				M.choose_name(3, src.name, default)
 				if(M.real_name != default && M.real_name != orig_real)
 					phrase_log.log_phrase("name-[ckey(src.name)]", M.real_name, no_duplicates=TRUE)
+				if((M.job == "Clown") || (M.job == "Mime"))
```

Comment:
So this doesn't look voluntary... I'm not sure how you're planning to address imposing "friendship" between several unrelated players (on RP or not) but this doesn't seem like a good idea.